lms/djangoapps/course_api/blocks/utils.py

```diff
@@ -0,0 +1,23 @@
+"""
+   Utils for Blocks
+"""
+from openedx.core.djangoapps.discussions.models import (
+    DiscussionsConfiguration,
+    Provider,
+)
+
+
+def filter_discussion_xblocks_from_response(response, course_key, request_from_mobile=False):
+    """
+    Removes discussion xblocks if request is from mobile and provider is openedx
```

Comment:
Docstring should emphasize on the purpose of the method, not when the method is called. We can skip the `request from mobile` part, and emphasize what the function actually does!